bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -565,7 +559,7 @@ protected void submitCallback(int code) {
                 }
             }
             LOG.error("Read of ledger entry failed: L{} E{}-E{}, Heard from {}. First unread entry is {}",
-                    new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHosts, firstUnread });
+                    new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHostsBitSet, firstUnread });
```

Comment:
I would still like to get the bookie inet address as we use it extensively for debugging. With this we get the index only right?